sites/public/src/components/shared/MapControl.tsx

```diff
@@ -0,0 +1,87 @@
+import { GoogleMap } from "@react-google-maps/api"
```

Comment:
I'm getting the same error that Ines is.  I think we're both using `yarn dev` vs `yarn build && yarn start`.  You need to add `import React from "react"` to this file to resolve the error